README.md

```diff
@@ -2,9 +2,13 @@
 ### Required
 #### Ponder and answer these questions as you work through the other material for this week.
 - What is a framework?
+An abstraction that provides application-specific software that can be selectively changed by users.
```

Comment:
A foundation for application development. Like the foundation of a home. It holds everything together